web/src/components/profileEdit/ProjectCardEdit.tsx

```diff
@@ -132,90 +194,32 @@ const ProjectCardEdit: React.FC<IProjectCardEditProps> = (props) => {
-                                            )}
-                                        </Field>
-                                    </Label>
+                                    <Input {...input} id="project-other" />
                                 </Flex>
+                                {meta.error && meta.touched && <Label as="span" style={{ position: "absolute", bottom: "0" }} variant="errorLabel" >{meta.error}</Label>}
                             </Flex>
-                        ))}
-                        <Field name="project-other" validate={validator.mustBeValidComponentOthers} defaultValue={''} initialValue={profileDetails.other}>
-                            {({ input, meta }) => (
-                                <Flex pb="25px" style={{ position: "relative" }}>
-                                    <Label htmlFor="project-other">Other:</Label>
-                                    <Flex flex="1 1 auto" justifyContent="flex-end">
-                                        <Input {...input} id="project-other" />
-                                    </Flex>
-                                    {meta.error && meta.touched && <Label as="span" style={{ position: "absolute", bottom: "0" }} variant="errorLabel" >{meta.error}</Label>}
-                                </Flex>
-                            )}
-                        </Field>
-                        {!pendingEditRequest && isProvisioned ? (
-                            //@ts-ignore
-                            <StyledFormButton style={{ display: 'block' }} >Request Update</StyledFormButton>
-                            ) : (
-                                <>
-                                    {/* @ts-ignore */}
-                                    <StyledFormDisabledButton style={{ display: 'block' }}>Request Update</StyledFormDisabledButton>
-                                    <Label as="span" variant="errorLabel" >Not available due to a {isProvisioned ? 'Update' : 'Provision'} Request</Label>
-                                </>
-                            )
-                        }
-                    </form>
-                )}
-            </Form>
-        </>
+                        )}
+                    </Field>
+                    {!hasPendingEdit && isProvisioned ? (
+                        //@ts-ignore
+                        <StyledFormButton style={{ display: 'block' }} >Request Update</StyledFormButton>
+                    ) : (
+                            <>
+                                {/* @ts-ignore */}
+                                <StyledFormDisabledButton style={{ display: 'block' }}>Request Update</StyledFormDisabledButton>
+                                <Label as="span" variant="errorLabel" >Not available due to a {isProvisioned ? 'Update' : 'Provision'} Request </Label>
+                            </>
+                        )
+                    }
+                </form>
+            )}
```

Comment:
Note for later: This block is identical between ProjectCardEdit and ContactCardEdit, only additions for QuotaCardEdit is use of term 'Quota' instead of 'Update' and it has `onClick={handleSubmit}`. This could likely be refactored to reduce the duplication.

Reply:
sounds good - can refactor